modules/vba_documents/app/controllers/vba_documents/v0/reports_controller.rb

```diff
@@ -8,17 +8,23 @@ class ReportsController < ApplicationController
       skip_before_action(:authenticate)
       before_action :validate_params
 
-      MAX_REPORT_SIZE = 100
+      MAX_REPORT_SIZE = 1000
       ID_PARAM = 'ids'
 
       def create
-        statuses = VBADocuments::UploadSubmission.refresh_and_get_statuses!(params[ID_PARAM])
-        render json: statuses,
+        statuses = VBADocuments::UploadSubmission.where(guid: params[ID_PARAM])
```

Comment:
we still probably want to set a limit here so that they can't put something like 10K ids and the payload returned becomes to much for us to serve.